rules/phpunit/src/Rector/SpecificMethod/AssertFalseStrposToContainsRector.php

```diff
@@ -71,6 +71,9 @@ public function refactor(Node $node): ?Node
         }
 
         $firstArgumentValue = $node->args[0]->value;
+        if ($firstArgumentValue instanceof StaticCall) {
```

Comment:
Again, should be `! $firstArgumentValue instanceof FuncCall`
Always go for stricter condition.

Reply:
I tried and it broke the tests.

Reply:
How?

Reply:
Can't put more time reporting here, sorry.

Try it out yourself locally (applying the change you're suggesting) and you should get the broken tests too.

Reply:
Alright. Let's give it a go now, and wait for more issues